client/electron/index.ts

```diff
@@ -264,6 +264,31 @@ function updateTray(status: TunnelStatus) {
       ...menuTemplate,
     ];
   }
+  // macOS: Add Connect/Disconnect menu item
+  if (process.platform === 'darwin') {
```

Comment:
We don't use Electron for macOS, so this change will not work. It has to be in the Cordova/Apple code.